src/LLVM_intrinsics.jl

```diff
@@ -775,8 +775,9 @@ for (fs, c) in zip([HORZ_REDUCTION_OPS_FLOAT, HORZ_REDUCTION_OPS_INT],
                    [FloatingTypes,            IntegerTypes])
     for f in fs
         f_red = Symbol("reduce_", f)
+        ff_base = Base.libllvm_version < v"12" ? "experimental.vector.reduce." : "vector.reduce."
```

Comment:
This could be declared outside the function (with a different name).

Reply:
Alternatively, I could replace this with
```julia
ff_base = string(Base.libllvm_version < v"12" ? "experimental.vector.reduce." : "vector.reduce.", $(QuoteNode(f)))
```
For the negligible improvement in compilation time from not concatenating inside the `@generated` function body.

Reply:
I prefer code that is the easiest to read. Compile time differences will be negligible, and there will not be any run time difference. Since there is an existing, similar check outside the function, I thought it would be good to either have all version-dependent checks in the same place. Anything else I leave to you.